### ISSUE TITLE
Deploy on Netlify fails

### ISSUE THREAD

**Issue by scottwalstead** [greglobinski/gatsby-starter-hero-blog#6]
Error: ./src/components/Seo/index.js
1:12:54 PM:   Module not found: Error: Cannot resolve 'file' or 'directory' ./Seo in /opt/bu  ild/repo/src/components/Seo
1:12:55 PM:   resolve file
1:12:55 PM:     /opt/build/repo/src/components/Seo/Seo doesn't exist
1:12:55 PM:     /opt/build/repo/src/components/Seo/Seo.js doesn't exist
1:12:55 PM:     /opt/build/repo/src/components/Seo/Seo.jsx doesn't exist
1:12:55 PM:   resolve directory
1:12:55 PM:     /opt/build/repo/src/components/Seo/Seo doesn't exist (directory default file  )
1:12:55 PM:     /opt/build/repo/src/components/Seo/Seo/package.json doesn't exist (directory   description file)
1:12:55 PM:   [/opt/build/repo/src/components/Seo/Seo]
1:12:55 PM:   [/opt/build/repo/src/components/Seo/Seo.js]
1:12:55 PM:   [/opt/build/repo/src/components/Seo/Seo.jsx]
1:12:55 PM:    @ ./src/components/Seo/index.js 5:11-27

Should the Seo file exist?

**Comment by gouravsood** [comment]
Works for me after I add required environment variables following https://dev.greglobinski.com/gatsby-starter-hero-blog blog. 

See this: https://compassionate-kowalevski-449a27.netlify.com/

**Comment by scottwalstead** [comment]
I added a .env and it didn't resolve the issues.

```
GOOGLE_ANALYTICS_ID=""
ALGOLIA_APP_ID=""
ALGOLIA_SEARCH_ONLY_API_KEY=""
ALGOLIA_ADMIN_API_KEY=""
ALGOLIA_INDEX_NAME=""
FB_APP_ID=""
```

**Comment by greglobinski** [comment]
@scottwalstead 
"Should the Seo file exist?" - absolutely it should.

**Comment by scottwalstead** [comment]
@greglobinski Thanks for the reply. I commented out the Algolia section for now until I get the time to set that up so the build is passing.

It's odd that my local build passes with Seo.jsx (`yarn build`) no problem but Netlify fails.

**Comment by scottwalstead** [comment]
@greglobinski I just tried renaming the file from .jsx to .js locally after I noticed that was the only difference and sure enough, it passed the Netlify build. Then I came to the site to check, and looks like you updated it about a week ago.

https://github.com/greglobinski/gatsby-starter-hero-blog/commit/ba49871807b1aa33fd6653c58fa55a485725c0fa

Thanks for fixing it. I think this issue can be closed out.
